doc/nrf/releases_and_maturity/release-notes-3.0.0-preview2.rst

```diff
@@ -0,0 +1,1124 @@
+  * The :ref:`installation` page with the following:
+
+    * The :ref:`installing_vsc` section with a list valid for both development environments.
+      The list now includes nRF Util as an additional requirement for :ref:`west runner <programming_selecting_runner>` for the |nRFVSC|, and the Windows-only requirement to install SEGGER USB Driver for J-Link for both development environments.
+    * The command-line instructions now use the ``nrfutil sdk-manager`` command instead of the ``nrfutil toolchain-manager`` command.
+      You can read more about the new command in the `nRF Util documentation <sdk-manager command_>`_.
+
+
+Board support
+=============
+
+* Removed support for the nRF52810 Desktop Mouse board (``nrf52810dmouse/nrf52810``).
+
+Build and configuration system
+==============================
+
+* Removed support for the deprecated multi-image builds (parent-child images) functionality.
+  All |NCS| projects must now use :ref:`sysbuild`.
+  See :ref:`child_parent_to_sysbuild_migration` for an overview of differences with parent-child image and how to migrate.
+* Updated the default runner for the ``west flash`` command to `nRF Util`_ instead of ``nrfjprog`` that is part of the archived `nRF Command Line Tools`_.
+  For more information, see the :ref:`build system section in the v3.0.0 migration guide <migration_3.0_recommended>` and the :ref:`programming_selecting_runner` section on the programming page.
+
+Bootloaders and DFU
+===================
+
+|no_changes_yet_note|
+
+Developing with nRF91 Series
+============================
+
+|no_changes_yet_note|
+
+Developing with nRF70 Series
+============================
+
+* Removed support for storing the nRF70 firmware patches in external flash without the :ref:`partition_manager`, as mentioned in :ref:`ug_nrf70_developing_fw_patch_ext_flash`.
+
+Developing with nRF54L Series
+=============================
+
+* Added HMAC SHA-256 with a 128-bit key type to KMU, as detailed in the :ref:`ug_nrf54l_crypto_kmu_supported_key_types` documentation section.
+
+Developing with nRF54H Series
+=============================
+
+* Removed the note on installing SEGGER USB Driver for J-Link on Windows from the :ref:`ug_nrf54h20_gs` page and moved its contents to the `nRF Util prerequisites`_ documentation.
+  The Windows-only requirement to install the SEGGER USB Driver for J-Link is now mentioned in the :ref:`installing_vsc` section on the :ref:`installation` page.
+
+Developing with nRF53 Series
+============================
+
+|no_changes_yet_note|
+
+Developing with nRF52 Series
+============================
+
+|no_changes_yet_note|
+
+Developing with Thingy:91 X
+===========================
+
+|no_changes_yet_note|
+
+Developing with Thingy:91
+=========================
+
+|no_changes_yet_note|
+
+Developing with Thingy:53
+=========================
+
+|no_changes_yet_note|
+
+Developing with PMICs
+=====================
+
+|no_changes_yet_note|
```

Comment:
The Developing with nPM2100 doc went in and the changelog item is in main. Should it be included here?